.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+image: leanprovercommunity/mathlib:gitpod
+
+vscode:
+  extensions:
+    - jroesch.lean
+
+tasks:
+  - command: . /home/gitpod/.profile && scripts/get-cache.sh
```

Comment:
This is a `lean-liquid` script, isn't it?

Reply:
aah, that's a good point, I guess. I'll copy it over for now.

Reply:
Actually, I removed the call to that script. I don't think we have olean caches for the tutorials project, and maybe we don't need them either.